content/tutorials/manage-the-rippled-server/configuration/run-rippled-as-a-validator.md

```diff
@@ -20,11 +20,15 @@ To run a `rippled` server as a validator, complete the following tasks:
 
 2. [Enable validation on your `rippled` server](#enable-validation-on-your-rippled-server).
 
-Beyond this, you can optionally complete the following tasks that help build trust in your validator:
+3. [Connect your validator to the network using public hubs](#connect-to-the-network-using-public-hubs).
+
+At this point, you are likely running your `rippled` server as an untrusted validator. It's worth noting that running an untrusted validator can also be helpful to the overall health of the network. Untrusted validators help set the standard that trusted validators are measured against. For example, if a trusted validator is disagreeing with a lot of untrusted validators, that might indicate a problem.
```

Comment:
I don't like the 'untrusted validator' language. I understand what you are trying to convey, but I think we should just leave it at validators.